src/hooks/useViewport/useViewport.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { renderHook, act } from '@testing-library/react-hooks';
+import { fireEvent, wait } from '@testing-library/react';
+import { useViewport } from './useViewport';
+
+describe('useViewport', () => {
```

Comment:
what do you think about addind a test that checks that the resize method is called only once when the hook is wrapped in the Provider? too much tests?

Reply:
I'll give it a go 👍🏻